PAY-5859: Payment Services drop-in documentation for Summit (#170)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/place-order.mdx

  compact
  options={[
    ['Option', 'Type', 'Req?', 'Description'],
    ['disabled', 'boolean', 'No', 'Disables the Place Order button.'],

>     ['handleValidation', 'function', 'No', 'A function that performs validation checks and returns a boolean indicating whether the validation passed; in case it returns false, the order will not be placed.'],

Reviewer: ```suggestion
    ['handleValidation', 'function', 'No', 'A function that performs validation checks and returns a boolean indicating whether the validation passed. If it returns false, the order will not be placed.'],
```
